src/node/interfaces.cpp

```diff
@@ -653,24 +654,25 @@ class ChainImpl : public Chain
     }
     void getPackageLimits(unsigned int& limit_ancestor_count, unsigned int& limit_descendant_count) override
     {
-        limit_ancestor_count = gArgs.GetIntArg("-limitancestorcount", DEFAULT_ANCESTOR_LIMIT);
-        limit_descendant_count = gArgs.GetIntArg("-limitdescendantcount", DEFAULT_DESCENDANT_LIMIT);
+        limit_ancestor_count = limit_descendant_count = 0;
+        if (!m_node.mempool) return;
```

Comment:
in 95430b611c6562845143669e88e07fcde770d1af "node/ifaces: Use existing MemPoolLimits"

I see your "Question for Reviewers: Is the default for limit_ancestor_count and limit_descendant_count in getPackageLimits correct?" in the commit message. To clarify, are you asking whether this has a behavior change, or are you asking if the change is better?

IIUC This returns limits = 0 when there is no mempool, where it would have previously returned the results of querying gArgs / defaults. This makes logical sense to me: if there's no mempool, then there's no mempool limits. It seems the one use for `getPackageLimits` is in `SelectCoins`, and the wallet understands it can still make an unconfirmed transaction (i.e. anc/desc count = 1) so it's fine: https://github.com/bitcoin/bitcoin/blob/874529665c1c326fc86fc0d0d6c3512fab087ab8/src/wallet/spend.cpp#L509-L511

I guess a future `-nomempool` option would just need to be consistent with this.

Reply:
Fixed in 4825207fc6c15fce4cc1cb9b14a66a51f3b5a47b, feel free to reopen if still a problem!